src/modules/createComponent/templates/component.tailwind.mustache

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react'
+
+interface Props {
+  className?: string
+  children?: ReactNode
```

Comment:
petition to pass the styled prop through by default, because it doesn't hurt anyone and instantly makes us more compatible with other libraries